src/components/project/project-step-change.resolver.ts

```diff
@@ -0,0 +1,19 @@
+import { Parent, ResolveField, Resolver } from '@nestjs/graphql';
+import { mapSecuredValue } from '../../common';
+import { Loader, LoaderOf } from '../../core';
+import { SecuredUser, UserLoader } from '../user';
+import { ProjectStepChange } from './dto';
+
+@Resolver(ProjectStepChange)
+export class ProjectStepTransitionResolver {
+  @ResolveField(() => SecuredUser)
+  async user(
+    @Parent() stepChange: ProjectStepChange,
+    @Loader(UserLoader) users: LoaderOf<UserLoader>
+  ): Promise<SecuredUser | null> {
+    if (!stepChange.user) {
+      return null;
+    }
```

Comment:
Secured types should not be nullable. Exposing null when there is no user, means they know that I value is missing even if they aren't supposed to be able to read it. That security probably doesn't matter much here, but I'm just trying to be consistent.
```suggestion
  ): Promise<SecuredUser> {
```
`mapSecuredValue` handles the value being null too.